### DISCUSSION TITLE
I have the same question how do I add my own cert? I'm running **Ubuntu 18.04** and **modoboa 1.17.0**

### DISCUSSION THREAD

**Discussion by Catospye** [modoboa/modoboa#2380] [General]
I have the same question how do I add my own cert? I'm running **Ubuntu 18.04** and **modoboa 1.17.0**

_Originally posted by @Catospye in https://github.com/modoboa/modoboa/issues/1697#issuecomment-942733803_

**Comment by tonioo** [comment]
Adding a custom certificate is a manual operation, there is no shortcut for that right now. You can copy your certificate files in the a secured directory on your server (by default, it is generally in /etc/ssl) and update nginx, postfix and dovecot configuration files to replace the generated certificate by yours.

**Reply by Catospye** [reply]
Im fully aware that there's no button or shortcut that can. I would like to know where the config files are for Nginx, postfix, and dovecot.

**Reply by tonioo** [reply]
/etc/nginx/site-enabled, /etc/postfix/main.cf, /etc/dovecot/conf.d/10-ssl.conf
